load/loader.go

```diff
@@ -4,22 +4,22 @@ import (
 	"bufio"
 	"flag"
 	"fmt"
+	"log"
 	"math"
 	"os"
 	"sync"
 	"sync/atomic"
 	"time"
-	"log"
 )
 
 const (
-	// DefaultBatchSize is the default size of batches to be inserted
+	// defaultBatchSize - default size of batches to be inserted
 	defaultBatchSize = 10000
 	defaultReadSize  = 4 << 20 // 4 MB
 
-	// WorkerPerQueue is the value to have each worker have its own queue of batches
+	// WorkerPerQueue - whether each worker has its own queue of batches
```

Comment:
`// WorkerPerQueue is the value for assigning each work its own queue of batches`

Reply:
Ok, agreed

Reply:
Updated as **workER**